src/main/kotlin/com/ampnet/blockchainapiservice/controller/VerificationController.kt

```diff
@@ -0,0 +1,31 @@
+package com.ampnet.blockchainapiservice.controller
+
+import com.ampnet.blockchainapiservice.model.response.GenerateVerificationMessageResponse
+import com.ampnet.blockchainapiservice.service.VerificationService
+import com.ampnet.blockchainapiservice.util.WalletAddress
+import mu.KLogging
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class VerificationController(private val verificationService: VerificationService) {
+
+    companion object : KLogging()
+
+    @PostMapping("/verification/{walletAddress}/generate")
+    fun generateVerificationMessage(
+        @PathVariable walletAddress: String
+    ): ResponseEntity<GenerateVerificationMessageResponse> {
```

Comment:
not related to this PR but,
we have to figure out how the client is going to consume this response object to sign the message.
I propose we use the [RLP encoding](https://eth.wiki/fundamentals/rlp) to encode all of the fields of this response on the client side before signing the message